pylint/reporters/text.py

```diff
@@ -35,8 +35,11 @@
 from pylint.reporters.ureports.text_writer import TextWriter
 
 if TYPE_CHECKING:
+    from pylint.lint import PyLinter
     from pylint.reporters.ureports.nodes import Section
 
+ColorMappingDict = Dict[str, Tuple[Optional[str], ...]]
```

Comment:
Not exactly what I had in mind 😅
The idea was to change the style definition from `Optional[str]` to `Tuple[str, ...]`. E.g.
```py
# old
("magenta", "bold, italic")

# new
("magenta", ("bold", "italic"))
```

Reply:
😂
Let me change that. Then it actually works with a namedtuple as well!